### PR TITLE
Update global image info and change file name

### LINKED ISSUE #2095: Update ‘Global’ image credit link and information
### Overview
We need to correct the [image credit Name link](https://www.hackforla.org/credits/) and credit information for the ‘Global’ image to the specific link and accurate information, because it currently has incorrect information and leads to a generic Noun Project link.
### Action Items
- [x] Replace the generic Noun Project link for the image titled “Global” in the [credits page](https://www.hackforla.org/credits/) with [this specific link](https://thenounproject.com/search/?q=impact&i=1710368) instead in the ‘Name’ category 
- [x] Change the credit Name of the icon from Global to “Environmental Impact”
- [x] Change the credit Artist of the icon from Flatart to “Made x Made”
### Resources/Instructions
[Credit.yml template wiki](https://github.com/hackforla/website/wiki/credit.yml-file-template)
[Credits page](https://www.hackforla.org/credits/)
[‘Environmental Impact’ (Global) correct Image link](https://thenounproject.com/search/?q=impact&i=1710368)

### PR DESCRIPTION
**Pull request by Zhu-Joseph**
Fixes #2095 

### What changes did you make and why did you make them ?

  - Replaced generic Noun Project link with [specific link](https://thenounproject.com/search/?q=impact&i=1710368)
  - Update title from Global to Environmental Impact
  - Update artist to Made x Made
  - Changed name of global.yml file to environmental-impact.yml

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Before-Global

</details>

<details>
<summary>Visuals after changes are applied</summary>

After-Global
aching_Files)

</details>

### CODE REVIEW

**Review by macho-catt — CHANGES_REQUESTED** [review]
Hi Joseph. For the most part, changes look good. However, you swapped title-link and provider-link. It should be the other way around. You can reference another yml file in the code (like care.yml).

File: _data/internal/credits/environmental-impact.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-title: Global
+title: Environmental Impact
 title-link: https://thenounproject.com/
 content: Image
 used-in: About
-artist: Flatart
+artist: Made x Made
 provider: Noun Project
-provider-link: https://thenounproject.com/
+provider-link: https://thenounproject.com/search/?q=impact&i=1710368
```
Comment (macho-catt) [inline _data/internal/credits/environmental-impact.yml]: This should stay to https://thenounproject.com/

File: _data/internal/credits/environmental-impact.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Global
+title: Environmental Impact
 title-link: https://thenounproject.com/
```
Comment (macho-catt) [inline _data/internal/credits/environmental-impact.yml]: Title link should be: https://thenounproject.com/search/?q=impact&i=1710368

File: _data/internal/credits/environmental-impact.yml (hunk 1 above)
Comment (Zhu-Joseph) [reply _data/internal/credits/environmental-impact.yml]: Hi Matt, I swapped and updated the links. They should be correct now.

File: _data/internal/credits/environmental-impact.yml (hunk 2 above)
Comment (Zhu-Joseph) [reply _data/internal/credits/environmental-impact.yml]: Hi Matt, I swapped and updated the links. They should be correct now.

**Review by macho-catt — APPROVED** [review]
LG. Good job!

**Pull request merged**